config/initializers/license.rb

```diff
@@ -5,3 +5,7 @@
 License = LagoUtils::License.new(Rails.application.config.license_url)
 
 License.verify unless Rails.env.test?
+
+if Rails.env.development?
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://rubystyle.guide#if-as-a-modifier)